### PR TITLE
Grid styling

### PR DESCRIPTION
**Pull request by Maclenn77**
We created a clone of YouTube page, using HTML and CSS Grid Layout. We added video, images, buttons, and we used semantic tags, reusable CSS to style the position of elements. For icons, we use and external source from Google Apis. 

We validated the code through W3C validator for CSS and HTML.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,26 +1,299 @@
+/* Create a three columns grid in the body position*/
+.wrapper {
+	display: grid;
+	grid-template-columns: 325px 325px 325px;
+	grid-template-rows: 100px 360px auto 200px auto;
+	grid-column-gap: 10px;
+	grid-row-gap: 10px;
+	/*	width: 100%;
+	height: 100%; */
```
Comment (euqueme) [inline css/style.css]: remove all commented code

File: css/style.css [hunk 2]
```diff
@@ -1,26 +1,299 @@
...
+
+/* Create a blue bar that represents video advance
+
+#blue-hr {
+	border: 1px solid #167ac6;
+	width: 150px;
+	position: relative;
+	top: 9px;
+	left: 70px;
+}
```
Comment (euqueme) [inline css/style.css]: Remove this commented Id #blue-hr

File: css/style.css [hunk 3]
```diff
@@ -1,26 +1,299 @@
...
+	padding-right: 5px;
+}
+
+/* Style like and dislike icons in the main position
+
+#like-dislike {
+	position: relative;
+	left: 500px;
+	top: -30px;
+}
```
Comment (euqueme) [inline css/style.css]: remove this commented id #like-dislike

File: index.html [hunk 4]
```diff
@@ -7,23 +8,83 @@
...
 <body>
+  <!-- A grid is create with the properties defined in the css file -->
+
+  <div class="wrapper">
+
+    <!-- Create a navbar in the header position with the properties defined in the css -->
+
+    <nav class="box-1">
+        <button type="button" class="menu"><i class="material-icons">dehaze</i> </button>
+        <img alt="YouTube Logo" src="images/yt-logo.jpg" width="100" height="128" class="logo">
```
Comment (euqueme) [inline index.html]: move the **width** and **height** selectors to a class in your CSS file

File: index.html [hunk 5]
```diff
@@ -7,23 +8,83 @@
...
+
+        <input type="text" placeholder="Search..." class="search">
+        <button type="button" class="material-icons search-icon">search</button>
+
+        <img alt="Twitter picture" src="images/twitter.jpg" height="30" width="40" id="twitter-img">
+        <img alt="Loupe picture" src="images/loupe.jpg" height="30" width="40" id="loupe-img">
+        <img alt="Photo-camera picture" src="images/photo-camera.jpg" height="30" width="40" id="photo-camera-img">
```
Comment (euqueme) [inline index.html]: move the **width** and **height** selectors to a class in your CSS file lines #24 to #26

File: index.html [hunk 6]
```diff
@@ -7,23 +8,83 @@
...
+    </nav>
+
+    <!-- Create a box in the main position with the properties defined in the css -->
+
+    <main class="box-2">
+
+      <iframe width="650" height="360" src="https://www.youtube.com/embed/V74l_zS1x8E"
```
Comment (euqueme) [inline index.html]: move the **width** and **height** selectors to a class in your CSS file

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @Maclenn77 and @markonikovic 

### Good job, your project looks professional and the youtube clone looks promising :v:  

### I have some changes still for you to make

## Page layout

I see you've used the http://archive.ph/Bss88 page and it's okay but this page has a ruler, the original page should be layered out taking 100% of the screen width like [this page](https://www.youtube.com/watch?v=V74l_zS1x8E) so let's fix it 

- Yours

image

- Original

image

## Sidebar

- [x] the title and views for the video thumbnails should be at the right and not at the bottom

- Yours

image

- Original

image

## General

* [x] I have found commented code in your CSS file please remove it I have put comments bellow where you can find them :arrow_double_down: 
* [x] I also have found inline styling in your HTML file, please remove it and add all style selectors in the CSS file you can only use ID's and classes in your HTML file :arrow_double_down: 

### That's it, team! :raised_hands: 

### Please :pray: do the changes :memo:  and submit back for a code review when you are done :rocket: 

Happy Coding! :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Comment by majovanilla** [comment]
Hey @Maclenn77  and @markonikovic  , 👋

## Good job on implementing the requested changes on this project! 👍

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- check your [HTML in the validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fgithub.com%2FMaclenn77%2FProject-Embeding-Videos%2Fblob%2Fgrid-styling%2Findex.html). You have several errors and warnings. :warning: 
Screenshot from 2020-02-21 11-55-27

- After implementing the previous required changes, your page looks a little different from the original one, especially with the positioning of some icons and elements. Please change them to look like the real youtube page :smile: 

In your page, the embedded video doesn't have the appropriate size and positioning. I should be aligned to the right, next to the suggested videos:

Screenshot from 2020-02-21 12-07-28

Original:

Screenshot from 2020-02-21 12-07-35

In your page the line that separates the video title is forcing the icons to move to the right side and below. Also, the suscribe button is in the opposite side.

Screenshot from 2020-02-21 11-54-28

Original page:

Screenshot from 2020-02-21 11-54-35

### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- Change the size of your nav icons and the youtube logo to avoid losing proportion. This will make them look better.

Screenshot from 2020-02-21 12-04-15

- Add some description to the suggested videos to make your page look closely to the original one.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes



You can find me on Social Media below 👇 . Remember to say hi!

[GitHub](https://github.com/majovanilla) [Twitter](https://twitter.com/MajoVanilla) [LinkedIn](https://www.linkedin.com/in/majoreyesparroquin/)

**Comment by SofiBretz** [comment]
### Changes required ♻️
### Hello team 🌟

Great work on this project, but we need to fix the issues below before moving on ⬇️

- [ ] Consider the positioning of the icons and elements, please make this look like the original page (Icons: 'likes', 'share', 'save'.. has to be in one single line as the original page)






- [x] Consider that the embedded video doesn't have the appropriate size and positioning. I should be aligned to the right, next to the suggested videos.





- [ ] Consider that the subscribe button is on the opposite side, please put it on the right as the original page.





[OPTIONAL]

- [ ] You are still missing the comment section. You can include only one or two comments to display it.

- [ ] Please add a demo link in your readme file.

_**After making the required changes please submit a new code review request**_
Good luck! 🍀 You are almost there! 💪

SofiBretz.

**Comment by Maclenn77** [comment]
Hi, Sofi! 

We commented @Marijo by Slack that the page should look like this version of YouTube website: http://archive.ph/Bss88, not like the current one. So, she said that these changes weren't required:

>     * [ ]  Consider the positioning of the icons and elements, please make this look like the original page (Icons: 'likes', 'share', 'save'.. has to be in one single line as the original page)
> 
> 
> 
> 
> 
> 
>     * [ ]  Consider that the subscribe button is on the opposite side, please put it on the right as the original page.
> 
> 
> 

We fixed the other problems. Greetings, and thx!

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Hi Team,
## Project looks amazing. Congratulation :+1: Approved.
I will ask you for one more thing, `node modules` folder should never be present in your repository, so please delete it. You can always use `.gitignore` file to prevent it from commiting said folder next time. 
Then go ahead and merge to master.

_Goodluck in your next project_ *Happy coding*

**Pull request merged**